src/plugins/auth/drop-user-session.js

```diff
@@ -0,0 +1,5 @@
+function dropUserSession () {
```

Comment:
remove this file and add the `dropUserSession` function to `auth/user-session.js`